clcache.py

```diff
@@ -1334,7 +1334,7 @@ def processDirect(cache, outputFile, compiler, cmdLine, sourceFile):
             origCmdLine = cmdLine
```

Comment:
I think if you insist on the term 'copy', you could also just define `def copyList(xs): return list(xs)` but at this point I think it's safe to assume that anybody who's somewhat familiar with Python knows that if you do `list(xs)` you create a new list object. ;-)